src/tests/Myr.test.js

```diff
@@ -444,6 +452,54 @@ describe("Component Animations", () => {
         expect(el).toHaveProperty("animation__shrink");
     });
 
+    it("it should add startEvent to the cursor", () => {
+        myr.reset();
+        myr.setAnimationTrigger("none");
+        expect(myr.startEvent).toEqual("");
+        myr.setAnimationTrigger("click");
+        expect(myr.startEvent).toEqual("click");
+        myr.setAnimationTrigger();
+        expect(myr.startEvent).toEqual("");
+        myr.setAnimationTrigger("mouseEnter");
+        expect(myr.startEvent).toEqual("mouseenter");
+        myr.setAnimationTrigger("mouseLeave");
+        expect(myr.startEvent).toEqual("mouseleave");
+
+        //check for stable model with bad values
+        myr.setAnimationTrigger("wrongArgument");
+        expect(myr.startEvent).toEqual("mouseleave");
+        myr.setAnimationTrigger(0);
+        expect(myr.startEvent).toEqual("mouseleave");
+        myr.setAnimationTrigger({});
+        expect(myr.startEvent).toEqual("mouseleave");
+    });
+
+    it("it should add delay to the cursor", () => {
+        myr.reset();
+        myr.setDelay(1000);
+        expect(myr.delay).toEqual(1000);
+        myr.setDelay("a");
+        expect(myr.delay).toEqual(1000);
+        myr.setDelay({});
+        expect(myr.delay).toEqual(1000);
+    });
+
+    it("should only reset the animation cursor property", () => {
```

Comment:
One thing to keep in mind for testing is side effects. We want to make sure that when we reset the animation cursor, we're only resetting the animation properties. Here we also don't change anything, so we don't know if they're actually being reset.